beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -3867,10 +3829,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
 
         if let Err(e) = self.store.do_atomically_with_block_and_blobs_cache(ops) {
             error!(
-                self.log,
-                "Database write failed!";
-                "msg" => "Restoring fork choice from disk",
-                "error" => ?e,
+                msg= "Restoring fork choice from disk",
```

Comment:
I'm so sorry but I can't help myself :smiling_face_with_tear: 
```suggestion
                msg = "Restoring fork choice from disk",
```